src/access/tests/TestAccessControl.cpp

```diff
@@ -820,260 +820,260 @@ struct CheckData
 constexpr CheckData checkData1[] = {
     // Checks for implicit PASE
     { .subjectDescriptor = { .fabricIndex = 0, .authMode = AuthMode::kPase, .subject = kPaseVerifier0 },
-      .requestPath       = { .cluster = 1, .endpoint = 2 },
+      .requestPath       = { .cluster = 1, .endpoint = 2, .requestType = RequestType::kAttributeReadRequest },
```

Comment:
Why not leave as unknown? 

Unless ARL is hooked-up and implemented (which it isn't in this PR), all the tests should still pass with the current cases having `unknown`.

I only expect future NEW tests to test ARL, AFTER setting an ARL over top of the non-ARL test data.

Reply:
The updated contract to existing ACL code is that requestType is required to be set per Boris' request/comment or it will fail the Check.  I think it makes sense to validate this argument is provided correctly along with this change.